tracer/build/_build/CodeGenerators/CallTargetsGenerator.cs

```diff
@@ -0,0 +1,743 @@
+                    switch (namedArgument.Name)
+                    {
+                        case nameof(InstrumentAttributeProperties.AssemblyName):
+                            assemblyName = namedArgument.Argument.Value?.ToString();
+                            break;
+                        case nameof(InstrumentAttributeProperties.AssemblyNames):
+                            assemblyNames = GetStringArray(namedArgument.Argument.Value);
+                            break;
+                        case nameof(InstrumentAttributeProperties.IntegrationName):
+                            integrationName = namedArgument.Argument.Value?.ToString();
+                            break;
+                        case nameof(InstrumentAttributeProperties.TypeName):
+                            typeName = namedArgument.Argument.Value?.ToString();
+                            break;
+                        case nameof(InstrumentAttributeProperties.TypeNames):
+                            typeNames = GetStringArray(namedArgument.Argument.Value);
+                            break;
+                        case nameof(InstrumentAttributeProperties.MethodName):
+                            methodName = namedArgument.Argument.Value?.ToString();
+                            break;
+                        case nameof(InstrumentAttributeProperties.ReturnTypeName):
+                            returnTypeName = namedArgument.Argument.Value?.ToString();
+                            break;
+                        case nameof(InstrumentAttributeProperties.MinimumVersion):
+                            minimumVersion = namedArgument.Argument.Value?.ToString();
+                            break;
+                        case nameof(InstrumentAttributeProperties.MaximumVersion):
+                            maximumVersion = namedArgument.Argument.Value?.ToString();
+                            break;
+                        case nameof(InstrumentAttributeProperties.ParameterTypeNames):
+                            parameterTypeNames = GetStringArray(namedArgument.Argument.Value);
+                            break;
+                        case nameof(InstrumentAttributeProperties.CallTargetType):
+                            //callTargetType = (namedArgument.Argument.Value as INamedTypeSymbol)?.ToDisplayString();
+                            break;
+                        case nameof(InstrumentAttributeProperties.CallTargetIntegrationKind):
+                            integrationKind = namedArgument.Argument.Value as int?;
+                            break;
+                        case nameof(InstrumentAttributeProperties.InstrumentationCategory):
+                            instrumentationCategory = (InstrumentationCategory)(namedArgument.Argument.Value as uint?).GetValueOrDefault();
+                            break;
+                        default:
+                            hasMisconfiguredInput = true;
+                            break;
```

Comment:
Should we throw here, so we can see the "incorrect" values 🤔 

In the source generator we had to be careful to accept "invalid" code, but I don't think we need to do that here, and it's better to just throw, as it suggests something is very wrong aand out of sync

```suggestion
                            throw new InvalidOperationException($"Integration type  '{type}' has an unknown attribute value: '{namedArgument.Name}'. The Nuke generator code may need updating."); 
```